SU-HSDO/suhumsci
# Stn 653  spotlight color dropdown

# READY FOR REVIEW

## Summary
Added option on the Spotlight paragraph type to select a global gray background color (#D9D7D2) or the 
 theme's default Spotlight background color.

View of select list options:
<img width="755" alt="Screen Shot 2022-02-16 at 9 16 26 AM" src="https://user-images.githubusercontent.com/25614178/154287682-79ae55ca-e763-4f98-a534-d917558f7e41.png">

View of results in Colorful/Cardinal:
<img width="1214" alt="Screen Shot 2022-02-16 at 9 55 02 AM" src="https://user-images.githubusercontent.com/25614178/154291256-d67a1461-ced0-419f-846e-79a50fd3c615.png">

View of results in Colorful/Mountain:
<img width="1217" alt="Screen Shot 2022-02-16 at 9 55 21 AM" src="https://user-images.githubusercontent.com/25614178/154291314-ede643a6-64dc-435f-bf17-6bd79c0bfdee.png">

Jira Card: https://sparkbox.atlassian.net/jira/software/c/projects/STN/boards/80?modal=detail&selectedIssue=STN-653

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. Create a new page and add a Spotlight paragraph type to that page.
3. Under the "Background Color" select list, select "Default background" and verify that the theme's default spotlight background appears.
4. Under the "Background Color" select list, select "Medium Gray background" and verify that the global medium gray background #D9D7D2 appears.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
